src/visualizer/vertex/Vertex.hpp

```diff
@@ -0,0 +1,28 @@
+//
+// Created by stefan on 21.07.2023.
+//
+
+#ifndef AGE_VERTEX_HPP
+#define AGE_VERTEX_HPP
+
+#include <QPainter>
+#include <QWidget>
+
+namespace age::visualizer {
+class Vertex : public QWidget {
+  Q_OBJECT
+public:
+  explicit Vertex(int x, int y, QWidget* pParent) noexcept;
+
+protected:
+  void mousePressEvent(QMouseEvent*) override;
+  void paintEvent(QPaintEvent*) override;
```

Comment:
Name the parameters of these two functions
```suggestion
  void mousePressEvent(QMouseEvent* pEvent) override;
  void paintEvent(QPaintEvent* pEvent) override;
```